airbyte-config/init/src/main/resources/seed/destination_specs.yaml

```diff
@@ -3414,7 +3414,7 @@
     supported_destination_sync_modes:
     - "overwrite"
     - "append"
-- dockerImage: "airbyte/destination-redshift:0.3.28"
+- dockerImage: "airbyte/destination-redshift:0.3.30"
```

Comment:
This should also be `31` right?

Reply:
This will be updated by CI job after image will be published.